frontend/apps/reader/modules/readermenu.lua

```diff
@@ -184,6 +186,20 @@ function ReaderMenu:setUpdateItemTable()
     self.tab_item_table = MenuSorter:mergeAndSort("reader", self.menu_items, order)
 end
 
+function ReaderMenu:exitOrRestart(callback)
+    self:onTapCloseMenu()
+    UIManager:scheduleIn(0.1, function()
```

Comment:
Yup, I specifically wrote `nextTick` to replace all the `scheduleIn(0.1)` hacks. There are still multiple `scheduleIn(0.1)` hacks that haven't been cleaned up yet.